dev/io.openliberty.org.apache.myfaces.4.0_fat/build.gradle

```diff
@@ -39,7 +39,15 @@ dependencies {
     requiredLibs project(':io.openliberty.org.apache.commons.logging')
     requiredLibs project(':io.openliberty.org.apache.xercesImpl')
     requiredLibs 'xml-apis:xml-apis:1.4.01'
-//  mojarra40 'org.glassfish:jakarta.faces:4.0.0'
+    requiredLibs('org.eclipse.jetty:jetty-io:9.2.15.v20160210') { force = true }
```

Comment:
what does force = true mean?

Reply:
It is a way to manage transitive dependencies. `htmlunit` and `neko-htmlunit` both have a dependency on `org.eclipse.jetty.websocket:websocket-client` which has its own dependencies.  They have dependencies on different versions (and use slightly different APIs)  both versions need to be available and so you need to force gradle to download and use both.